builder/image/templates/builder

```diff
@@ -113,7 +105,7 @@ if [ ! -f Dockerfile ]; then
     BUILD_OPTS+=' deis/slugbuilder'
 
     # build the application and attach to the process
-    JOB=$(eval '$(parse-string "${BUILD_OPTS[@]}")' )
+    JOB=$(${BUILD_OPTS[@]})
```

Comment:
not necessarily. `JOB=${BUILD_OPTS[@]} ` means that you are setting $JOB to the literal array, whereas `JOB=$(${BUILD_OPTS[@]})` means that JOB is set to the output of that command. The following is the equivalent though:

```
JOB=`${BUILD_OPTS[@]}\`
```